frontend/src/components/ViewLinks.vue

```diff
@@ -1,77 +1,51 @@
 <template>
   <b-navbar-nav>
+    <!-- <li class="play-to-earn top-nav-links">
+      <P2EButton mainText="Play-to-earn" route="play-to-earn" />
+    </li> -->
+
+    <li class="character top-nav-links" >
+      <router-link v-if="!stakeOnly" :to="{ name: 'plaza' }" exact class="nav-link">
+        <div class="icon"> <img src="../assets/new-ui/new-plaza-icon.png" class="new-ui-links-icon"></div>
+        <div class="link-text">Character</div>
+      </router-link>
+    </li>
+
+   <li v-if="!stakeOnly" >
+      <router-link :to="{ name: 'blacksmith' }" exact class="nav-link">
+        <div class="icon"><img src="../assets/new-ui/new-blacksmith-icon.png" class="new-ui-links-icon"></div>
+        <div class="link-text">Blacksmith</div>
```

Comment:
Missing translations in places that already had translations... Look at the deleted code